addons/udes_stock_picking_batch/models/stock_picking.py

```diff
@@ -1,4 +1,4 @@
-from odoo import models, fields, api
+from odoo import models, fields, api, _
 from .common import PRIORITY_GROUPS
 from odoo.exceptions import ValidationError
 
```

Comment:
I think the story asks to add a function called get_batch_for_pick? is that still required?

Reply:
get_batch_for_pick lives in the workflows controller class in udes_11 in udes_workflows/controllers/common, and calls batch_to_user. I think when we port workflows this method will be ported doesn't seem right to port it as part of this story.

Reply:
No problem there as long as TL and PO are happy.